src/histolab/filters/image_filters_functional.py

```diff
@@ -21,60 +21,61 @@
 from functools import reduce
 
 import numpy as np
+import PIL
+import PIL.ImageOps
```

Comment:
Is it necessary to import also PIL.ImageOps?

Reply:
it seems necessary. Importing only PIL and calling `PIL.ImageOps.function` seems not possible. @ernestoarbitrio any clues?

Reply:
Offline he said he has no clues. I'll leave as it is for the moment